code/modules/reagents/chemistry/reagents.dm

```diff
@@ -32,21 +32,26 @@
 	var/addiction_stage3_end = 30
 	var/addiction_stage4_end = 40
 	var/overdosed = 0 // You fucked up and this is now triggering its overdose effects, purge that shit quick.
-	var/self_consuming = FALSE
-	var/metabolizing 		= FALSE
-	var/invisible			= FALSE //Set to true if it doesn't appear on handheld health analyzers.
+	var/self_consuming = FALSE  //I think this uhhh, makes weird stuff happen when metabolising, but... doesn't seem to do what I think, so I'm gonna leave it.
 	//Fermichem vars:
-	var/purity = 1 						//How pure a chemical is from 0 - 1.
-	var/addProc = FALSE 				//If the chemical should force an on_new() call
-	var/turf/loc 			= null 				//Should be the creation location!
+	var/purity 				= 1 		//How pure a chemical is from 0 - 1.
+	var/cached_purity		= 1
+	var/turf/loc = null 				//Should be the creation location!
 	var/pH = 7							//pH of the specific reagent, used for calculating the sum pH of a holder.
-	var/SplitChem			= FALSE		//If the chem splits on metabolism
-	var/ImpureChem 			= "fermiTox"// What chemical is metabolised with an inpure reaction
-	var/InverseChemVal 		= 0.25		// If the impurity is below 0.5, replace ALL of the chem with InverseChem upon metabolising
-	var/InverseChem 		= "fermiTox"// What chem is metabolised when purity is below InverseChemVal, this shouldn't be made, but if it does, well, I guess I'll know about it.
-	var/DoNotSplit			= FALSE		// If impurity is handled within the main chem itself
-	var/OnMobMergeCheck 	= FALSE 	//Call on_mob_life proc when reagents are merging.
-
+	//var/SplitChem			= FALSE		//If the chem splits on metabolism
+	var/ImpureChem						// What chemical is metabolised with an inpure reaction
+	var/InverseChemVal 		= 0			// If the impurity is below 0.5, replace ALL of the chem with InverseChem upon metabolising
+	var/InverseChem 					// What chem is metabolised when purity is below InverseChemVal, this shouldn't be made, but if it does, well, I guess I'll know about it.
+	var/metabolizing = FALSE
+	var/reagentFlags //REAGENT_DEAD_PROCESS, REAGENT_DONOTSPLIT, REAGENT_ONLYINVERSE, REAGENT_ONMOBMERGE, REAGENT_INVISIBLE, REAGENT_FORCEONNEW, REAGENT_SNEAKYNAME
+	//REAGENT_DEAD_PROCESS  	Calls on_mob_dead() if present in a dead body
+	//REAGENT_DONOTSPLIT		Do not split the chem at all during processing
+	//REAGENT_ONLYINVERSE		Only invert chem, no splitting
+	//REAGENT_ONMOBMERGE		Call on_mob_life proc when reagents are merging.
+	//REAGENT_INVISIBLE			Doesn't appear on handheld health analyzers.
+	//REAGENT_FORCEONNEW		Forces a on_new() call without a data overhead
+	//REAGENT_SNEAKYNAME    	When inverted, the inverted chem uses the name of the original chem
+	//REAGENT_SPLITRETAINVOL 	Retains initial volume of chem when splitting
```

Comment:
Replace these comments with a single directing the reader to the file containing those flags (e.g. see __DEFINES/reagents.dm for information on each the flags). Less maintenance trouble.

Reply:
I moved it into a readme I'll fill up as I go along.